build/ci.go

```diff
@@ -284,12 +284,13 @@ func buildFlags(env build.Environment, staticLinking bool, buildTags []string) (
 
 func doTest(cmdline []string) {
 	var (
-		dlgo     = flag.Bool("dlgo", false, "Download Go and build with it")
-		arch     = flag.String("arch", "", "Run tests for given architecture")
-		cc       = flag.String("cc", "", "Sets C compiler binary")
-		coverage = flag.Bool("coverage", false, "Whether to record code coverage")
-		verbose  = flag.Bool("v", false, "Whether to log verbosely")
-		race     = flag.Bool("race", false, "Execute the race detector")
+		dlgo         = flag.Bool("dlgo", false, "Download Go and build with it")
+		arch         = flag.String("arch", "", "Run tests for given architecture")
+		cc           = flag.String("cc", "", "Sets C compiler binary")
+		coverage     = flag.Bool("coverage", false, "Whether to record code coverage")
+		verbose      = flag.Bool("v", false, "Whether to log verbosely")
+		race         = flag.Bool("race", false, "Execute the race detector")
+		stateTestDir = "tests/spec-tests"
```

Comment:
Is there any reason for definining it here? If you move it into `downloadSpecTestFixtures`, you will have all related to spec-tests collected there, which might be nicer?